### DISCUSSION TITLE
How to uninstall a mod?

### DISCUSSION THREAD

**Discussion by danielpza** [Senth/minecraft-mod-manager#129] [Q&A] [answered]
I don't see any command for it, I don't want to remove it manually because it doesn't modify the db file. Is there an recommended way to remove a mod?

**Comment by Senth** [comment, marked as answer]
HI Daniel,

The recommended way to remove it is to remove it manually. The DB file will be modified the next time `mcman` runs: If the mod isn't present when running `mcman` it will set the mod as `inactive` in the DB. Meaning `mcman` won't try to update it. So the DB is always in sync with the actual `mods` folder.

The reason for keeping a record in the DB is that it saves all the settings of site-slug etc., so that if you temporarily disable a mod (by moving it or renaming it to something else than `.jar` at the end, it should keep the configuration).

Hope this helps 🙂
